src/gui/modals/EffectSelectDialog.cpp

```diff
@@ -126,7 +127,7 @@ EffectSelectDialog::EffectSelectDialog(QWidget* parent) :
 
 	m_filterEdit = new QLineEdit(this);
 	connect(m_filterEdit, &QLineEdit::textChanged, this, [this](const QString &text) {
-		m_model.setFilterRegExp(QRegExp(text, Qt::CaseInsensitive));
+		m_model.setFilterRegularExpression(QRegularExpression(text, QRegularExpression::CaseInsensitiveOption));
```

Comment:
@messmerd I could narrow the CI build fail to this one line. 
from [this article](https://forum.freecad.org/viewtopic.php?t=65089&start=30)
> It was the QRegularExpression-taking "setFilterRegularExpression()" function from QSortFilterProxyModel that I had remembered biting me (that was only added in Qt 5.12).

Reply:
So is there a chance #7015 might fix it?

Reply:
Yes, 7015 will fix this.

Reply:
7015 wouldn't fix the MinGW build unfortunately, so the problem would remain

Reply:
But atleast linux will get fixed.
If anyone got any suggestions, please leave it here.

Reply:
Try falling back to QRegExp if we're on Qt5?
```suggestion
// TODO: Cleanup when we don't support Qt5 anymore
#if (QT_VERSION >= QT_VERSION_CHECK(6, 0, 0))
		m_model.setFilterRegularExpression(QRegularExpression(text, QRegularExpression::CaseInsensitiveOption));
#else
		m_model.setFilterRegExp(QRegExp(text, Qt::CaseInsensitive));
#endif
```

Reply:
That's viable, but needs more change than that one line. Will attempt soon.

Reply:
Nvm, i saw it wrong.